Classes7.3/DeviceAbstractClass.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Classes7._3
+{
+    public abstract class DeviceAbstractClass
+    {
+        public double numberOfPixelsInCamera;
```

Comment:
This field is not used in Printer class. You can move this as property to ITakePhoto interface

Reply:
The same with paperWidth, paperHeight

Reply:
Fixed both 😅